### DISCUSSION TITLE
[Docs] Menu is missing on mobile

### DISCUSSION THREAD

**Discussion by netcoding87** [chakra-ui/chakra-ui#2355] [General] [answered]
Hey, I am missing the hamburger icon or another option to open the menu on the mobile on [https://next.chakra-ui.com/docs](https://next.chakra-ui.com/docs/getting-started). Is this intended behaviour or is here something broken?

![image](https://user-images.githubusercontent.com/54022880/97407098-7b08d880-18fa-11eb-9216-4c195861fb5f.png)

**Comment by ljosberinn** [comment, marked as answer]
The docs aren't finished yet, so this is "intended". The search will also return eventually.
